include/OpenSpaceToolkit/Physics/Environment/Gravitational/Spherical.hpp

```diff
@@ -31,9 +31,9 @@ class Spherical : public Model
    public:
     /// @brief              Constructor
     ///
-    /// @param              [in] aGravitationalParameter A gravitational parameter
+    /// @param              [in] parameters A set of gravitational parameters
 
-    Spherical(const Derived& aGravitationalParameter);
+    Spherical(const Model::Parameters& parameters);
```

Comment:
Naming wise, I recommend:

```suggestion
    Spherical(const Model::Parameters& aParameterSet);
```

Reply:
Agreed! I was toying with aSetOfParameters but I like this a lot better.